backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -242,6 +246,7 @@ def create_provider_privileges(
         jurisdiction_postal_abbreviations: list[str],
         license_expiration_date: date,
         compact_transaction_id: str,
+        existing_privileges: list[dict],
```

Comment:
Keyboard Navigation (using VoiceOver):  `/Account`

* Functionality of the "hide/unhide password" icon is very unclear using voice navigation.
* I get no input about the incorrect and correct password requirements, or why I can't click the submit button (which still seems to be enabled, even if there is a form validation error?)